### DISCUSSION TITLE
Multiple abstracts

### DISCUSSION THREAD

**Discussion by roaldarbol** [typst/typst#4712] [Q&A] [answered]
Hi y'all! Is there a way to generate multiple abstracts? This is a nice feature when stringing together papers for a thesis. 😊 E.g. if multiple documents are imported into a main document?
Cheers!

**Comment by Enivex** [comment, marked as answer]
There's no built-in abstract in typst, so this would be entirely on whatever template you're using. If you make your own it should be no problem

**Reply by roaldarbol** [reply]
Ah, that's my bad, thanks! I've been using MyST Markdown to interface with Typst, and they have an abstract field, so I just assumed it was inherited from Typst. I'll direct my questions there!
